.templates/README.md.erb

```diff
@@ -0,0 +1,56 @@
+# CIC Bootcamp
+## Introduction
+This course is intended to teach good practices around Infrastructure-as-Code development. This course contains a set of materials which are designed to teach you how to write tests for and automate the release of the things you build.
+
+To undertake this course you will need a computer which you have full control over (i.e not your employer's desktop build) - It will need to be on a network with unrestricted access to the Internet (not behind locked-down corporate firewalls) as you will need to download and install a few components before you can begin.
+
+Topics covered in this material:
+```
+<%= command_output('cic track list') %>
+```
+
+To remove the need for dependencies on an external IaaS (Infrastructure as a Service) solution in order to participate in this course, this course has been designed to use Docker containers to simulate real computers.
+
+The course is being actively built out so be sure to `git pull` regularly to get new exercises as they appear.
+
+## Participants
+### Environment Requirements
+For this material you will need the following:
+ - A Bash compatible environment
+ - git
+ - Docker - If you don't have it already, follow the instructions on the [Docker website](https://docs.docker.com/install/#next-release) to install it on your system
+ - Editor of your choice
+
+### Setup
+1. create a .netrc file in your home directory with your credentials for github in it.
+  Your entry should look something like this:
+  ```
+  machine api.github.com
+      login username
+      password password
+  ```
+2. Fork this repo
```

Comment:
Maybe a link to the github docs on forking would be an idea hear... People won't all necessarily be au-fait with git based workflows... https://help.github.com/articles/fork-a-repo/